crates/solvers/src/tests/dex/partial_fill.rs

```diff
@@ -120,12 +120,11 @@ async fn tested_amounts_decrease() {
                 "sellToken": "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2",
                 "buyToken": "0xba100000625a3754423978a60c9317c58a424e3D",
                 "sellAmount": "16000000000000000000",
-                "buyAmount": "3641580551073046209760",
-                // Let's just assume 0 fee to not further complicate the math.
+                "buyAmount": "3630944624685908136768",
```

Comment:
This value changed for there to be enough room in the limit price to account for the surplus fee (the swap used to match exactly at the limit price).